src/components/PaginationV2/index.tsx

```diff
@@ -0,0 +1,178 @@
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import { useMemo } from 'react'
+import Button from '../Button'
+import Icon from '../Icon'
+import getPageNumbers from './getPageNumbers'
+
+const PageNumbersContainer = styled.div`
+  display: flex;
+  gap: ${({ theme }) => theme.space['2']};
+  margin: 0 ${({ theme }) => theme.space['2']};
+`
+
+const StyledPageSwitch = styled(Button)`
+  width: ${({ theme }) => theme.space['4']};
+  height: ${({ theme }) => theme.space['6']};
```

Comment:
please do not use space for sizing, it is reserved for spacing (padding, margin, etc)